txndata/constraints.lisp

```diff
@@ -391,7 +391,7 @@
 (defconstraint setting-priority-fee-per-gas (:guard (first-row-trigger))
   (if-zero TYPE2
            (eq! PRIORITY_FEE_PER_GAS (gas_price))
-           (eq! PRIORITY_FEE_PER_GAS (- (gas_price) BASEFEE))))
+           (eq! PRIORITY_FEE_PER_GAS (- GAS_PRICE BASEFEE))))
```

Comment:
So IIRC `GAS_PRICE` is the effective gas price of the transaction as computed either by using the transaction's own `GAS_PRICE` (type 0/1) or with the type 2 stuff, right ? and the `(gas-price)` refers to either the transaction's GAS_PRICE or the `MAX_PRIORITY_FEE` for type 2, something like this ? Both of them being data that is extracted from the TXN_RLP module